packages/clay-form/src/__tests__/__snapshots__/InputWithMultiSelect.tsx.snap

```diff
@@ -44,6 +44,11 @@ exports[`ClayInputWithMultiSelect renders as not valid 1`] = `
         className="form-control form-control-tag-group"
         onKeyDown={[Function]}
       >
+        <input
+          name=""
```

Comment:
I wanted to double check what would happen when using the component's default (empty string) name:

> If no name is specified, or name is empty, the input's value is not submitted with the form.

([Source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input))

So seems ok.

Reply:
Thanks for checking that out.